komsic/XChanger
# #169522021 Add The Button Component

#### What does this PR do?
Add the button component for easy reusability

#### Description of Task to be completed?
- set base style for all components and containers
- implement the button component
- add tests to affirm implementation

#### How should this be manually tested?
- clone the repo
- cd into it
- run ```yarn``` on cmd
- run ```yarn storybook``` to see the implemented repo

#### What are the relevant pivotal tracker stories?
[#169522021](https://www.pivotaltracker.com/story/show/169522021)
